app.py

```diff
@@ -31,6 +34,15 @@ def create_app():
     oauth_client = WebApplicationClient(client_id)
     login_manager.init_app(app)
 
+    app.logger.setLevel(os.environ['LOG_LEVEL'])
+
+    loggly_token = os.environ['LOGGLY_TOKEN']
+
+    if loggly_token is not None:
```

Comment:
`os.environ['LOGGLY_TOKEN']` will already have thrown if `LOGGLY_TOKEN` isn't in `os.environ`.

I think you want something like this, and get rid of `LOGGLY_TOKEN` in `.env.test`
```suggestion
    app.logger.setLevel(os.environ.get('LOG_LEVEL', 'DEBUG'))

    loggly_token = os.environ.get('LOGGLY_TOKEN')

    if loggly_token is not None:
```